ccpp/data/GFS_typedefs.meta

```diff
@@ -6957,14 +6957,14 @@
 [rh02max]
   standard_name = maximum_relative_humidity_at_2m_over_maximum_hourly_time_interval
   long_name = maximum relative humidity at 2m over maximum hourly time interval
-  units = %
+  units = 1
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
 [rh02min]
   standard_name = minimum_relative_humidity_at_2m_over_maximum_hourly_time_interval
   long_name = minumum relative humidity at 2m over maximum hourly time interval
-  units = %
+  units = 1
```

Comment:
May I ask what is the meaning of "units 1" and "units mixed"? Maybe you can see the units in NCEP grib2 product table as a reference? https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-1.shtml

Reply:
Relative humidity in the code is expressed between 0 and 1, i.e. as a fraction. If it was a real percentage, it would range from 0 to 100. See the actual code in ccpp-physics.

Note also we do have `percent` as a valid unit for such quantities. The `%` symbol is not acceptable as a unit in the CCPP, it causes all sorts of problem with parsing code (also because members of derived data types have a percent in the name, e.g. `foo%bar`). We could use `fraction` instead of `1` for relative humidity, but if so we need to go back to the CCPP framework developers and have them confirm.

Any variable that is a number without a physical unit can have unit `1`.

`mixed` is simply replacing `various` and is used in cases where a variable contains multiple units. For example, the grab-bag tracer array `qgrs` contains mass mixing ratios (`kg kg-1`), number concentrations `(kg-1)`, cloud amount (`fraction` or `1`, can't remember) etc. When the individual members of the `qgrs` array are passed, then the correct units can be used. But when the entire array is passed, we need to use `mixed` (was `various` until now).

Lastly, note that CCPP units do not follow the standard in https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-1.shtml. If they do match, that is great, and we should strive for consistency where possible. But CCPP units follow CF conventions and `udunits`, with the extensions defined in https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst (as approved by the CCPP framework developer team).

Reply:
Comparing the grib2 doc with what we have I see a nearly 100% match. The only difference is that we use `percent` instead of `%` for real percentages. Some units may differ from scheme to scheme (e.g. Thompson MP defines cloud effective radii in micron), but this is why we have automatic unit conversions - this way the host models can keep their units and still use the physics as-is.